crypto/crypto.go

```diff
@@ -46,7 +46,7 @@ const DigestLength = 32
 
 var (
 	secp256k1N, _  = new(big.Int).SetString("fffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141", 16)
-	secp256k1halfN = new(big.Int).Div(secp256k1N, big.NewInt(2))
```

Comment:
This happens exactly once per execution of geth. I see no reason to shave off some nanoseconds here

Reply:
reverted